scrape.py

```diff
@@ -91,16 +94,38 @@ def parse_instance(tr):
         i.ebs_only = True
     i.ebs_optimized = totext(cols[10]).lower() == 'yes'
     i.network_performance = totext(cols[4])
+    i.enhanced_networking = totext(cols[11]).lower() == 'yes'
     print "Parsed %s..." % (i.instance_type)
     return i
 
+def _rindex_family(inst2family, details):
```

Comment:
This might be better defined inside the scrape_families() function since it's only used there. This collection of scraping functions is getting kind of crazy... :)